bodhi/tests/server/services/test_updates.py

```diff
@@ -719,7 +719,7 @@ def test_provenpackager_request_privs(self, publish, *args):
     @mock.patch(**mock_valid_requirements)
     @mock.patch('bodhi.server.notifications.publish')
     def test_provenpackager_request_update_queued_in_ci(self, publish, *args):
-        """Ensure provenpackagers cannot request changes for any update which
+        """Ensure provenpackagers can request changes for any update which
```

Comment:
Isn't "cannot" correct here? The test seems to assert that the provenpackager is unable to do this.